[FIX] Disable start button

ipz234vas opened the pull request:
- Fix disabling sign-in button
- Fix sign-in form view
- Fix refresh token setting
- Add disabling start button while searching last session
- Add searchingLastSession field to state

ipz234vas committed:
[FIX] Disable start button

- Fix disabling sign-in button
- Fix sign-in form view
- Fix refresh token setting
- Add disabling start button while searching last session
- Add searchingLastSession field to state

mmv-vt231 commented on timetracker.client/src/components/layouts/AppLayout.tsx:
```diff
@@ -18,6 +19,7 @@ function AppLayout() {
     const location = useLocation();
 
     useEffect(() => {
+        dispatch(setSearchingLastSession(true));
```
change loading in getLastWorkSession

mmv-vt231 commented on timetracker.client/src/features/time-tracker/components/TimeTrackerHeader.tsx:
```diff
@@ -11,6 +11,7 @@ import { useAppSelector } from "@hooks/useAppSelector.ts";
 function TimeTrackerHeader() {
     const userId = useAppSelector(state => state.authentication.user?.id)
     const isTracking = useAppSelector(state => state.timeTracker.isTracking)
+    const SearchingLastSession = useAppSelector(state => state.timeTracker.SearchingLastSession)
```
rename to searchingLastSession

mmv-vt231 commented on timetracker.client/src/features/time-tracker/components/TimeTrackerHeader.tsx:
```diff
@@ -75,6 +75,7 @@ function TimeTrackerHeader() {
                             variant="ghost"
                             display="flex"
                             gap="2"
+                            isDisabled={SearchingLastSession}
```
rename to searchingLastSession

mmv-vt231 commented on timetracker.client/src/features/time-tracker/timeTrackerSlice.ts:
```diff
@@ -7,6 +7,7 @@ const initialState: TimeTrackerState = {
     workSessions: [],
     sessionId: null,
     isTracking: false,
+    SearchingLastSession: false,
```
rename to searchingLastSession

ipz234vas committed:
[FIX] Sign-in button loading

- Refactor sign-in button loading
- Rename SearchingLastSession to searchingLastSession
- Relocate setSearchingLastSession function call
